ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -0,0 +1,68 @@
+
+if have_tensorflow
+  tensor_filter_tfcore_lib_shared = shared_library('nnstreamer_filter_tensorflow',
+      [
+        'tensor_filter_tensorflow_core.cc',
+        'tensor_filter_tensorflow.c'
+      ],
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: [tf_dep, protobuf_dep, glib_dep, gst_dep, orc_dep, nnstreamer_dep],
+    include_directories: nnstreamer_inc,
+    install: true,
+    install_dir: filter_subplugin_install_dir
+  )
+
+  tensor_filter_tfcore_lib_static = static_library('nnstreamer_filter_tensorflow',
+      [
+        'tensor_filter_tensorflow_core.cc',
+        'tensor_filter_tensorflow.c'
+      ],
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: [tf_dep, protobuf_dep, glib_dep, gst_dep, orc_dep, nnstreamer_dep],
+    include_directories: nnstreamer_inc,
+    install: true,
+    install_dir: nnstreamer_libdir
+  )
+
+  tensor_filter_tfcore_lib = tensor_filter_tfcore_lib_shared
```

Comment:
line 27~33 and 60~67, unnecessary declaration

Reply:
Commit updated